internals/docs/guides/inspections.md

```diff
@@ -1,3 +1,114 @@
 ## Tizen.Inspections Guide
 
-TBD
\ No newline at end of file
+`Tizen.Inspections` is used to collect data from applications and services. This API allows you to receive inspection events as well as request inspection data on demand (ex. receive crash reports if a process in the system crashes).
+
+## Prerequisites
+
+Follow [get started](https://docs.tizen.org/application/dotnet/get-started/overview/) page to setup Visual Studio for .NET development.
+
+To enable your application to use `Tizen.Inspections`:
+
+1. To use `Tizen.Inspections` to collect inspectable data, the application must be signed with a platform level distributor certificate. Refer this [guide](https://docs.tizen.org/application/dotnet/tutorials/certificates/getting-the-certificates/) on how to sign an application with the custom certificate.
+
+2. To use the methods and properties of the `Tizen.Inspections` namespace, include the namespace in your application::
+
+    ```c#
+    Using Tizen.Inspections
+    ```
+
+## Receive inspection events
+
+`Tizen.Inspections` allows you to receive notification in case of abnormal situation occurs on the platform (ex. process crashes). To do so, you must subscribe to the `EventReceived` event:
+
```

Comment:
Not only abnormal situation. 
In diagnostic view, components (modules or applications) tries to publish  their status. 
Please, check ROS's diagnostic summary - http://wiki.ros.org/diagnostics

Thus, I think that following is better 

"Tizen.Inspections" allows to receive status notification published by the platform (at future, Tizen inpsections will support to receive status notification published by manufacturer's applications)

Reply:
Done.